packages/notion-app-enhanced-deb/notion-app-enhanced-deb.pacscript

```diff
@@ -0,0 +1,11 @@
+name="notion-app-enhanced-deb"
+pkgname="notion-app-enhanced"
+repology=("project: ${pkgname}")
+version="2.0.18-1"
+url="https://github.com/notion-enhancer/notion-repackaged/releases/download/v${version}/${pkgname}_${version}_amd64.deb"
+breaks="${pkgname}-app notion-app-deb notion-app-app"
```

Comment:
```suggestion
breaks="${pkgname}-bin notion-app-deb notion-app"
```

Reply:
The name of the other package is notion-app, the other "-app" is the suffix that indicates AppImage format. And they don't provide binary packages, so there's no need for the "-bin".